flocker/control/script.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script for starting control service server.
+"""
+
+from twisted.python.usage import Options
+from twisted.internet.endpoints import TCP4ServerEndpoint
+
+from .httpapi import create_api_service
+from ..common.script import (
+    flocker_standard_options, FlockerScriptRunner, main_for_service)
+
+
+@flocker_standard_options
+class ControlOptions(Options):
+    """
+    Command line options for ``flocker-control`` cluster management process.
+    """
+    optParameters = [
+        # XXX also choose where configuration will be stored
+        ["port", "p", 4523, "The port to listen on.", int],
+        ]
+
+
+class ControlScript(object):
+    """
+    A command to start a long-running process to control a Flocker
+    cluster.
+    """
+    def main(self, reactor, options):
+        # XXX also start persistence service
```

Comment:
And I guess the API service should get a reference to the persistence service?  Though it won't do anything with it yet... Perhaps it should?  Should this branch come with an implementation of the `get the desired configuration` API endpoint, perhaps?  It seems like that should be a pretty trivial addition to the other stuff that's going to happen and it might be a good way to demonstrate persistence (not the only way though - having the config survive process restarts would demonstrate it too, except I don't think there's any way to observe that it has survived right now ..)